tests/parm/input_global_parent.nml.IN

```diff
@@ -0,0 +1,410 @@
+&atmos_model_nml
+  blocksize = @[BLOCKSIZE]
+  chksum_debug = .false.
+  dycore_only = .false.
+  ccpp_suite = "@[CCPP_SUITE]"
+  
+/
+
+&diag_manager_nml
+  prepend_date = .false.
+  max_output_fields = @[MAX_OUTPUT_FIELDS]
+  
+/
+
+&fms_nml
+  clock_grain = 'ROUTINE'
+  domains_stack_size = @[DOMAINS_STACK_SIZE]
+  print_memory_usage = .false.
+  
+/
+
+&fms2_io_nml
+  netcdf_default_format = "netcdf4"
+/
+
+&fv_core_nml
+  layout = @[INPES],@[JNPES]
+  io_layout = 1,1
+  npx = @[NPX]
+  npy = @[NPY]
+  ntiles = @[NTILES]
+  npz = @[NPZ]
+  dz_min =  @[DZ_MIN]
+  grid_type = 0
+  make_nh = @[MAKE_NH]
+  fv_debug = .false.
+  range_warn = .true.
+  reset_eta = .false.
+  n_sponge = @[N_SPONGE]
+  n_zs_filter = @[N_ZS_FILTER]
```

Comment:
The current global workflow does not have line 40.

Reply:
There will be many new lines the global workflow doesn't have.

Reply:
When one is not running a nest, any new options should match the namelist defaults. They can be found here:

- fv_core_nml = FV3/atmos_cubed_sphere/model/fv_arrays.F90
- gfs_physics_nml = FV3/ccpp/data/GFS_typedefs.F90